addons/website_event_track_quiz/controllers/event_track_quiz.py

```diff
@@ -32,6 +33,8 @@ def event_track_quiz_submit(self, event_id, track_id, answer_ids):
         result = {
             'answers': {
                 answer.question_id.id: {
+                    'awarded_points': answer.awarded_points,
+                    'correct_answer': answer.question_id.correct_answer_ids.text_value,
```

Comment:
Why is it a "correct_answer_ids" and not a "correct_answer_id"?
If questions can only have a single correct answer, one record should be enough.

(And "answer.question_id.correct_answer_ids.text_value" makes it seem like we will get expected singleton errors)